README.md

```
@@ -95,6 +95,73 @@ nwg-dock -d -i 48 --mb 10 --hide-timeout 400 --opacity 75 --launch-animation -c
 nwg-dock --wm sway
 ```
 
+## Configuration file
+
+In addition to CLI flags, `nwg-dock` reads a TOML config file at:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)